index.js

```diff
@@ -7,31 +7,48 @@ module.exports = function(glob, opts, cb) {
   if (typeof opts === 'function') {
     cb = opts;
     opts = {};
+  } else {
+    if(!opts) opts = {};
   }
 
-  var watcher = gaze(glob, opts, function(err, rwatcher){
-    if (err) out.emit('error', err);
-    rwatcher.on('all', function(evt, path, old){
-      var outEvt = {type: evt, path: path};
-      if(old) outEvt.old = old;
-      out.emit('change', outEvt);
-      if(cb) cb(outEvt);
-    });
+  if(opts.ignoreInitial === undefined) opts.ignoreInitial = true;
+
+  var nomatch = true;
+
+  var watcher = chokidar.watch(glob, opts).on('all', function(evt, path, stats){
```

Comment:
this fn for mapping the events should be split out into a new fn like `chokidar2Gaze`